src/main/java/homework4/task1/Robot.java

```diff
@@ -0,0 +1,61 @@
+package homework4.task1;
+
+public class Robot {
+
+    public Direction getDirection() {
+       return this.getDirection(); // текущее направление взгляда
+    }
+
+    public int getX() {
+       return this.getX(); // текущая координата X
+    }
+
+    public int getY() {
+       return this.getY(); // текущая координата Y
+    }
+
+    public void turnLeft() {
+        // повернуться на 90 градусов против часовой стрелки
```

Comment:
write the implementation of this method
it will be useful for you: more practice - more experience